test/README.md

```diff
@@ -4,7 +4,7 @@
 - Have a file by the name `.env` in the root folder of this project.
 - This file should contain the below environment variables pointing to the various Service Bus resources
     - SERVICEBUS_CONNECTION_STRING
-    - QUEUE_NAME
+    - QUEUE_NAME (without sessions enabled)
```

Comment:
well... we eventually will have tests for sessions as well
So I would rather add a new variable called `SESSION_ENABLED_QUEUE_NAME`

Reply:
I agree -- But I think we need to call out that this queue doesn't have sessions enabled. If you do, the tests will fail.